include/zephyr/bluetooth/bluetooth.h

```diff
@@ -517,6 +525,60 @@ size_t bt_data_get_len(const struct bt_data data[], size_t data_count);
  */
 size_t bt_data_serialize(const struct bt_data *input, uint8_t *output);
 
+struct bt_le_local_features {
+	/**
+	 * @brief Local LE controller supported features.
+	 *
+	 * Refer to BT_LE_FEAT_BIT_* for values.
+	 * Refer to the BT_FEAT_LE_* macros for value comparionson.
+	 * See Bluetooth Core Specification 6.0, Vol 6, Part B, Section 4.6 table 4.10
```

Comment:
I believe we can leave out Core revision and Table number here. Older and newer revisions will have it in the same place:
```suggestion
	 * See Bluetooth Core Specification, Vol 6, Part B, Section 4.6.
```
I have no strong preference on this.

Reply:
You are probably correct. Looks like when sections are removed they are still kept and marked as "This section is no longer used", so perhaps we can consider section numbers static across versions.